dev/Materials/Acrylic/AcrylicBrush_rs2_themeresources.xaml

```diff
@@ -240,6 +240,17 @@
                 FallbackColor="#202020"
                 BackgroundSource="HostBackdrop"/>
             <SolidColorBrush x:Key="SystemControlDefaultBrighteningBrush" Color="#FFFFFF" Opacity="0.0419"/>
+            <local:AcrylicBrush x:Key="AccentAcrylicBackgroundFillColorDefaultBrush"
+                TintColor="{ThemeResource SystemAccentColorDark1}"
+                TintOpacity="0.8"
+                FallbackColor="{ThemeResource SystemAccentColorDark1}"
+                BackgroundSource="HostBackdrop"/>
+            <local:AcrylicBrush x:Key="AccentAcrylicBackgroundFillColorBaseBrush"
+                TintColor="{ThemeResource SystemAccentColorDark2}"
+                TintOpacity="0.8"
+                FallbackColor="{ThemeResource SystemAccentColorDark2}"
+                BackgroundSource="HostBackdrop"/>
+            <SolidColorBrush x:Key="AccentLayerFillColorDefaultBrush" Opacity="0.25" Color="{ThemeResource SystemAccentColorDark1}"/>
```

Comment:
Move this brush to Common_themeresources.xaml below x:Key="LayerFillColorTertiaryBrush"

Reply:
Should we also move the lighting brush too? It's also not an acrylic brush.

Reply:
Not sure what you mean by lighting brush

Reply:
Oh sorry, recalled the name incorrectly. I mean the `SystemControlDefaultBrighteningBrush` brush.